packages/core/cms/faststore/content-types.json

```diff
@@ -50,14 +50,62 @@
       }
     ]
   },
+  {
+    "id": "home",
+    "name": "Home",
+    "isSingleton": true,
+    "configurationSchemaSets": [
+      {
+        "name": "Settings",
+        "configurations": [
+          {
+            "name": "seo",
+            "schema": {
+              "title": "SEO",
+              "description": "Search Engine Optimization options",
+              "type": "object",
+              "widget": {
+                "ui:ObjectFieldTemplate": "GoogleSeoPreview"
+              },
+              "required": ["slug", "title", "description"],
+              "properties": {
+                "slug": {
+                  "title": "Path",
+                  "type": "string",
+                  "default": "/"
+                },
+                "title": {
+                  "title": "Default page title",
+                  "description": "Display this title when no other tile is available",
+                  "type": "string",
+                  "default": "FastStore"
+                },
+                "description": {
+                  "title": "Meta tag description",
+                  "type": "string",
+                  "default": "A beautifuly designed store"
+                },
+                "canonical": {
+                  "title": "Canonical url for the page",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        ]
+      }
+    ]
+  },
   {
     "id": "pdp",
     "name": "Product Page",
+    "isSingleton": true,
```

Comment:
Not sure if it is a singleton

Reply:
I think it should be a singleton, but when we add the multiple pdp/plp feature in the future, we can remove it.